### ISSUE TITLE
iColorFormat doesn't support RGBA texture?

### ISSUE THREAD

**Issue by grancia** [cisco/openh264#3056]
Hi,

I want to use OpenH264 to encode RGBA textures to h264. 
I'm able to see videoFormatRGBA in video type in library, but it looks only supporting I420 format according to welsEncoderExt.cpp like below. 
Currently it doesn't support RGBA texture format for encoding? 

```
int CWelsH264SVCEncoder::EncodeFrame (const SSourcePicture* kpSrcPic, SFrameBSInfo* pBsInfo) {

  if (kpSrcPic->iColorFormat != videoFormatI420)
    return cmInitParaError;
```

**Comment by huili2** [comment]
You are right. There is no plan to support other formats encoding except I420.
